kafka/datadog_checks/kafka/data/conf.yaml.example

```diff
@@ -18,725 +65,132 @@ instances:
     port: 9999
 
     ## @param user - string - optional
-    ## Username from the credentials needed to connect to the host.
+    ## User to use when connecting to JMX.
     #
-    # user: <USERNAME>
+    # user: <USER>
 
     ## @param password - string - optional
-    ## Password from the credentials needed to connect to the host.
+    ## Password to use when connecting to JMX.
     #
     # password: <PASSWORD>
 
-    ## @param process_name_regex - string - optional
-    ## Instead of specifying a host, and port. The agent can connect using the attach api.
-    ## This requires the JDK to be installed and the path to tools.jar to be set below in tools_jar_path parameter.
+    ## @param process_name_regex - string - optional - default: .*process_name.*
+    ## Instead of using a host and port, the Agent can connect using the attach API.
+    ## This requires the JDK to be installed and the path to tools.jar to be set below.
+    ## Note: It needs to be set when process_name_regex parameter is set
+    ## e.g. .*process_name.*
     #
     # process_name_regex: .*process_name.*
 
-    ## @param tools_jar_path - string - optional
-    ## Needs to be set when process_name_regex parameter is set.
+    ## @param tools_jar_path - string - optional - default: /usr/lib/jvm/java-8-openjdk-amd64/lib/tools.jar
+    ## The tool.jar path to be used with the `process_name_regex` parameter,
+    ## for example: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar
     #
-    # tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar
+    # tools_jar_path: /usr/lib/jvm/java-8-openjdk-amd64/lib/tools.jar
```

Comment:
Should that be 7 ?

Reply:
7 is EOL

Reply:
In fact java 8 is eol since may too

Reply:
Yup but we're mixing 7 and 8 in here

Reply:
and 9 and 10.....
11 is the current alive lts

Reply:
Sorry I wasn't clear, I meant here: https://github.com/DataDog/integrations-core/blob/e5cbedf5c08218fddb6148bb259a0ad7596e2663/kafka/datadog_checks/kafka/data/conf.yaml.example#L85-L89

Let's just be consistent between those lines :)

Reply:
yes, I'm _trying_ to submit a pr to fix the parent spec